kin_base/horizon.py

```diff
@@ -216,7 +215,7 @@ def __init__(self,
                                 # Events that dont have an id are not useful for us (hello/byebye events)
                                 # Save the last event id and retry time
                                 last_id = event.last_event_id
-                                retry = client._reconnection_time
+                                retry = client._reconnection_time.total_seconds()
```

Comment:
what's `reconnection_time`? and what's `total_seconds` ?

Reply:
```_reconnection_time``` is the time to sleep before renewing the connection, its in milliseconds so ```.total_seconds``` converts it

Reply:
👍